apps/trustlab/src/components/RichTypography.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-env browser */
+
+import isExternalUrl from "@/commons-ui/next/utils/isExternalUrl";
+import { RichTypography as CuiRichTypography } from "@commons-ui/core";
+import { useRouter } from "next/router";
+import React from "react";
+
+const RichTypography = React.forwardRef(function RichTypography(
```

Comment:
Isn't this already in both `@commons-ui/core` and `@commons-ui/next`? What are we adding/changing?